src/puppetlabs/puppetdb/command.clj

```diff
@@ -86,23 +101,78 @@
    "store report" (version-range 3 6)
    "deactivate node" (version-range 1 3)})
 
-;; ## Command parsing
+(defn- read-command-body
+  [body]
+  (if (= (class body) (class (byte-array 0)))
```

Comment:
Agreed, though since we also refer to the class in a couple of other contexts, I just added a utils/byte-array-class and used that everywhere.